backend/config.loader.php

```diff
@@ -1,15 +1,12 @@
 <?php
 
-if (
-    !empty($_SERVER['REQUEST_URI']) &&
-    substr($_SERVER['REQUEST_URI'], 0, 8) == '/branch/'
-) {
-    // for Branches
-    require_once __DIR__.'/../../../backend/config.php';
-} else if ( is_readable(__DIR__.'/config.php') ) {
-    // for Configured MASTER
+if ( is_readable(__DIR__.'/config.php') ) {
+    // for configured local
     require_once __DIR__.'/config.php';
+} else if ( is_readable(__DIR__.'/../../_backend/config.php') ) {
```

Comment:
The only problem I could possibly see with that is if we ever wanted to have the beta branches use a different config than master. Otherwise I'm good with using the configuration file in master.